helpers/apiData.js

```diff
@@ -0,0 +1,30 @@
+const api = require('../api');
+
+/* @deprecated */
+const getAllPaginatedData = async (req, service, qs) => {
```

Comment:
i would not prefer a deprecated function but instead why not let us export getApiData as getAllPagesFromApi or somewhat and remove the $limit from the request qs? Then everybody would know what happens...